backup.py

```diff
@@ -333,6 +353,32 @@ def main(argv):
 
             vm = vm[0]
 
+            # Get the Attachments Disk
+            disk_attachments = vms_service.vm_service(vm.id).disk_attachments_service().list()
+            disksBackup = []
```

Comment:
please use the python convention
https://peps.python.org/pep-0008/#function-and-variable-names
like:
```suggestion
            disks_backup = []
```